<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="SDGL691" name="Rediff WebSite Testing" tests="9" failures="1" timestamp="17 Sep 2014 11:47:29 GMT" time="58.294" errors="0">
  <testcase name="windowsTest" time="14.128" classname="com.rediff.testcases.QuickTestWinFF"/>
  <testcase name="testApp1" time="0.001" classname="com.rediff.testcases.RandomTest"/>
  <testcase name="testApp2" time="0.001" classname="com.rediff.testcases.RandomTest"/>
  <testcase name="testApp3" time="0.001" classname="com.rediff.testcases.RandomTest"/>
  <testcase name="linuxTest" time="33.683" classname="com.rediff.testcases.QuickTestLinFF"/>
  <testcase name="googleCheeseExample" time="8.983" classname="com.rediff.testcases.Selenium2ExampleST"/>
  <testcase name="googleMilkExample" time="8.718" classname="com.rediff.testcases.Selenium2ExampleST"/>
  <testcase name="test1_2" time="9.134" classname="com.rediff.testcases.Test1"/>
  <testcase name="test1_3" time="3.137" classname="com.rediff.testcases.Test1">
    <failure type="java.lang.AssertionError" message="expected [abc] but found [ab]">
      <![CDATA[java.lang.AssertionError: expected [abc] but found [ab]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:494)
	at org.testng.Assert.assertEquals(Assert.java:123)
	at org.testng.Assert.assertEquals(Assert.java:176)
	at org.testng.Assert.assertEquals(Assert.java:186)
	at com.rediff.testcases.Test1.test1_3(Test1.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- test1_3 -->
</testsuite> <!-- Rediff WebSite Testing -->
